src/redux/thunks/loadPolicies.ts

```diff
@@ -0,0 +1,220 @@
+import {createAsyncThunk} from '@reduxjs/toolkit';
+
+import {v4 as uuid} from 'uuid';
+
+import {AlertEnum} from '@models/alert';
+import {BasicPolicy, POLICY_VALIDATOR_MAP, Policy, PolicyConfig, ValidatorId} from '@models/policy';
+
+import {setAlert} from '@redux/reducers/alert';
+import {loadBinaryResource} from '@redux/services';
+
+import featureJson from '@src/feature-flags.json';
+
+import {loadPolicy} from '@open-policy-agent/opa-wasm';
+
+export const loadPolicies = createAsyncThunk<Policy[]>('main/loadPolicies', async (_, {dispatch}) => {
+  if (!featureJson.ResourceScanning) {
+    return [];
+  }
+
+  try {
+    const plugin = DEFAULT_TRIVY_PLUGIN;
+    const wasm = loadBinaryResource('policies/bundle/policy.wasm');
+
+    if (!wasm) {
+      throw new Error('policy not found');
+    }
+
+    const validatorId: ValidatorId = uuid();
+    const validator = await loadPolicy(wasm);
+    POLICY_VALIDATOR_MAP[validatorId] = validator;
+
+    return [{validatorId, metadata: plugin, config: DEFAULT_CONFIGURATION}];
+  } catch (err) {
+    dispatch(
+      setAlert({
+        title: 'Failed to load policy',
+        message: (err as Error).message,
+        type: AlertEnum.Warning,
+      })
+    );
+    return [];
+  }
+});
+
+const DEFAULT_CONFIGURATION: PolicyConfig = {enabled: true};
+
+const DEFAULT_TRIVY_PLUGIN: BasicPolicy = {
+  name: 'Default Trivy policies',
+  id: 'io.kubeshop.monokle.templates.default.policy',
+  author: 'kubeshop.io',
+  version: '0.1.0',
+  description: 'Default policies for Kubernetes resources.',
+  type: 'basic',
+  module: './policy.wasm',
+  rules: [
+    {
+      id: 'KSV003',
+      shortDescription: {
+        text: 'Default capabilities not dropped',
+      },
+      longDescription: {
+        text: 'The container should drop all default capabilities and add only those that are needed for its execution.',
+      },
+      helpUri: 'https://kubesec.io/basics/containers-securitycontext-capabilities-drop-index-all/',
+      help: {
+        text: "Add 'ALL' to containers[].securityContext.capabilities.drop.",
+      },
+      properties: {
+        severity: 'low',
+        entrypoint: 'appshield/kubernetes/KSV003/deny',
+        path: '$container.securityContext.capabilities.drop',
```

Comment:
The idea is that this can be a full path or work with special keywords. `$container` is necessary because one rule checks containers for `Pod`, `Deployment`, `Job` which have containers located at different locations. I notice all rules so far are actually on container so this is the most important to have but we can extend this later.